.github/workflows/build-notifications-image.yml

```diff
@@ -0,0 +1,31 @@
+name: Build Notifications Docker Image
+
+env:
+  IMAGE_NAME: notifications
+  AWS_REGION: eu-west-1
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  test:
+    name: Test Notifications
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+        with:
+          fetch-depth: '0'
+
+      - name: Build Docker Compose Stack...
+        run: |
+          cd ${GITHUB_WORKSPACE}/tests
+          docker-compose build
+
+      - name: Run Notifications Image Test
+        run: |
+          cd ${GITHUB_WORKSPACE}/tests
+          docker-compose up --abort-on-container-exit --exit-code-from tester tester
```

Comment:
Doesn't this need to run `tests/docker-test.sh`

Reply:
All the shell script is doing is running `docker-compose build` followed by the `docker-compose up` line as above - as I deliberately broke out the build and run stages for GH Actions (following the disappearing logs issue), the shell script isn't doing us any good here, other than maybe being a convenient way to run the tests on your local environment.

Reply:
I'd rather its the same process to run the tests regardless of whether you're running them locally or on GHA (for now they do the same thing but over time they can diverge). Now you've fixed the disappearing logs issue this can be a single step using the script can't it (which also follows the other similar Docker builds)